modules/ghlint-rules/src/test/kotlin/net/twisterrob/ghlint/rules/JobDependenciesRuleTest.kt

```diff
@@ -262,8 +270,9 @@ class JobDependenciesRuleTest {
 			val results = check<JobDependenciesRule>(file)
 
 			results shouldHave singleFinding(
-				"JobDependencyCycle",
-				"Job[test3] forms a dependency cycle: [test3].",
+				issue = "JobDependencyCycle",
+				message = "Job[test3] forms a dependency cycle: [test3].",
+				location = file("test3", 2),
```

Comment:
@timsearle what do you think about this syntax?

is it weird to use `operator invoke` here? if yes, any idea for a name?

Function defined in [locationOfNth.kt](https://github.com/TWiStErRob/net.twisterrob.ghlint/pull/320/files#diff-bf1987f171dd86cb0db359bd04e805ddee9d95ac887e7bb5c88f62e4274f834b) with tests at [LocationOfNthKtTest](https://github.com/TWiStErRob/net.twisterrob.ghlint/pull/320/files#diff-ccd07aff615611996006ad343030a19faf43da657d99022816eeb4fb531eebb3)

Reply:
https://github.com/TWiStErRob/net.twisterrob.ghlint/issues/330